docs/install/tarball.rst

```diff
@@ -9,6 +9,13 @@ This section of the documentation outlines how to use the release archives to
 install CrateDB. The walkthrough is suitable to install and run CrateDB on
 `Unix-like`_ systems, for example Linux and macOS.
 
+.. CAUTION::
+
+    You may experience performance issues when running releases from the public
+    archive on ARM-based macOS systems. For improved performance, we recommend
+    manually building CrateDB suited for macOS. Detailed instructions
```

Comment:
```suggestion
    manually building CrateDB suited for ARM-based macOS. Detailed instructions
```

Reply:
just a suggestion, it feels more clear to repeat that we are suggesting this for arm-based machines.